packages/client/components/NewMeetingTeamPickerAvatars.tsx

```diff
@@ -0,0 +1,72 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React, {useMemo} from 'react'
+import {useFragment} from 'react-relay'
+import {PALETTE} from '../styles/paletteV3'
+import defaultUserAvatar from '../styles/theme/images/avatar-user.svg'
+import getShuffledArr from '../utils/getShuffledArr'
+import {NewMeetingTeamPickerAvatars_team$key} from '../__generated__/NewMeetingTeamPickerAvatars_team.graphql'
+import Avatar from './Avatar/Avatar'
+import ErrorBoundary from './ErrorBoundary'
+
+const Container = styled('div')<{count: number}>(({count}) => ({
+  width: 44,
+  display: 'flex',
+  justifyContent: count > 1 ? 'flex-start' : 'center',
+  flexWrap: 'wrap',
+  bottom: count > 2 ? 4 : 0,
+  position: 'relative'
+}))
+
+const AvatarWrapper = styled('div')<{count: number}>(({count}) => ({
+  width: count > 1 ? 20 : 'auto',
+  height: count > 2 ? 20 : 'auto'
+}))
+
+const StyledAvatar = styled(Avatar)({
+  border: `2px solid ${PALETTE.SLATE_200}`
+})
+
+interface Props {
+  teamRef: NewMeetingTeamPickerAvatars_team$key
+}
+
+const NewMeetingTeamPickerAvatars = (props: Props) => {
+  const {teamRef} = props
+
+  const team = useFragment(
+    graphql`
+      fragment NewMeetingTeamPickerAvatars_team on Team {
+        teamMembers {
+          id
+          picture
+        }
+      }
+    `,
+    teamRef
+  )
+
+  const {teamMembers} = team
+
+  const randomAvatars = useMemo(() => {
+    const randomTeamMembers = getShuffledArr(teamMembers)
```

Comment:
+1 @igorlesnenko I'm assuming the goal here is to help the viewer recognize the team without using the team name. What do you think about:
- always showing the team lead, since they're probably the most recognizable user on the team ("Oh yeah, that's Igor's team")
- removing the viewer from the shuffle when the team size > 1 & they're not the team lead (that way, we can provide as much useful info as possible)

up to you, lgtm!

Reply:
I like putting the team lead first.
I would only hide the viewer if team size > 4, having only 1 head shown for a team of 2 would be confusing.

Reply:
This all make sense to me, I'll merge this PR for now and do changes in new issue